datamodels/2.x/authent-cas/dictionaries/zh_cn.dict.authent-cas.php

```diff
@@ -2,13 +2,10 @@
 /**
  * Localized data
  *
- * @copyright Copyright (C) 2010-2024 Combodo SAS
- * @license    https://opensource.org/licenses/AGPL-3.0
- * 
- */
-/**
- *
+ * @copyright   Copyright (C) 2013 XXXXX
```

Comment:
This is not good, please keep as it was:
```suggestion
 * @copyright Copyright (C) 2010-2024 Combodo SAS
```

Reply:
plz update en.dict first

Reply:
You changed in the Chinese version, Combodo normally uses a script that generates/updates the new dict items without changing the header in a file. They don't do a full copy.